catalog-info.yaml

```
@@ -0,0 +1,58 @@
+# Declare a Backstage Component that represents your application.
+---
+# yaml-language-server: $schema=https://json.schemastore.org/catalog-info.json
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: beats
+
```

should we add the description
```suggestion
  description: :tropical_fish: Beats - Lightweight shippers for Elasticsearch & Logstash 
```